src/plugins/intel_gpu/include/intel_gpu/runtime/internal_properties.hpp

```diff
@@ -174,6 +174,7 @@ static constexpr Property<std::vector<std::string>, ov::PropertyMutability::RW>
 static constexpr Property<bool, ov::PropertyMutability::RW> could_use_flashattn_v2{"GPU_COULD_USE_FLASHATTN_V2"};
 static constexpr Property<uint64_t, PropertyMutability::RW> dynamic_quantization_group_size_max{"GPU_DYNAMIC_QUANTIZATION_GROUP_SIZE_MAX"};
 static constexpr Property<bool, ov::PropertyMutability::RW> validate_output_buffer{"GPU_VALIDATE_OUTPUT_BUFFER"};
+static constexpr Property<float, ov::PropertyMutability::RW> pool_mem_utilization_threshold{"GPU_POOL_MEM_UTILIZATION_THRESHOLD"};
```

Comment:
Very minor comment, but how about GPU_MEM_POOL_UTIL_THRESHOULD? 
(because we call it memory pool, ...)